### DISCUSSION TITLE
Blinky Balancer and Ball Balancer Code

### DISCUSSION THREAD

**Discussion by Vinz1911** [pybricks/support#1199] [Q&A] [answered]
Hi @laurensvalk @dlech 

I saw this two cool balancer bot projects from you. Is the code for them already available? If yes, where can I find it, I did not saw it in the projects repo, so maybe I oversaw something ^_^

btw: is there any chance to buy from you one of this insane "run on pybricks" tiles ? 😍

greetings
Vinz

**Comment by laurensvalk** [comment, marked as answer]
Hey @Vinz1911 

> I saw this two cool balancer bot projects from you. Is the code for them already available?

Even if this took perhaps longer than expected, we can now say yes :smile: 

Click the images below for the instructions for the respective ball balancers. Up next, the regular balancers like the Blinky robot. Stay tuned!

[![balancer-mindstorms](https://github.com/pybricks/support/assets/12326241/71927b68-5fa5-496e-9c6c-ea80b9a5bbaa)](https://pybricks.com/project/lego-mindstorms-ball-balancer/)

[![spike-balancer](https://github.com/pybricks/support/assets/12326241/51e7dbe0-18f0-4e5d-bc0d-e828b90e2e26)](https://pybricks.com/project/lego-spike-ball-balancer/)

**Comment by inzoolo** [comment]
Hi Laurens,
i build teh balancer like you and put the programm on the lego spike hub. When i start the program, both motors rotate fast with same speed all the time. They dont react to the gyro.
can you tell me what i did wrong? I make all the steps you showed at the project page..
best regards, Andreas
